### PR TITLE
enable Codecov

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?
Re-enable Codecov from full GPU tests and add online badge

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by Borda** [comment]
@PyTorchLightning/core-contributors ^^

File: .codecov.yml [hunk 1]
```diff
@@ -2,9 +2,16 @@
 # Validation check:
 # $ curl --data-binary @.codecov.yml https://codecov.io/validate
 
+
+# https://docs.codecov.io/docs/codecovyml-reference
 codecov:
+  token: "d0fcb731-42af-465d-b680-c1ccdc824bdc"
```
Comment (luiscape) [inline .codecov.yml]: I think we are supposed to keep these tokens secret (we need to re-generate them if that's the case).

Given that you are using the token as an env var below (`$CODECOV_TOKEN`). Do you even need to configure it here?

Comment (Borda) [reply .codecov.yml]: no, it was just a duplicate and I forget to remove it... Done now :]

**Comment by Borda** [comment]
> I think that the codecov secret needs to be used as an env var instead of hard-coding it in the `codecov.yml` environment variable. If that is the case indeed, please re-generate the token and add it as an env var.

by my understanding this hardcoded token did not have any effect and just forget to remove it, done now, thx for the catch :] also new toked uploaded to Drone CI

**Pull request merged**